src/js/themes/hpe.js

```diff
@@ -379,11 +379,17 @@ const buildTheme = (tokens, flags) => {
       large.hpe.text?.[textSize]?.fontWeight || fallback.weight;
   });
 
-  textTheme.extend = ({ size: textSize, weight }) =>
-    !weight ? `font-weight: ${fontWeights[textSize]};` : '';
+  textTheme.extend = ({ size: textSize, weight }) => {
+    if (!weight) return `font-weight: ${fontWeights[textSize]};`;
+    if (weight === 'bold') return `font-weight: 500;`;
```

Comment:
I think it would be better to have this mapping from `bold` to `500` in the theme rather than hardcoded in a per-component way. Then this logic could actually be in base.js